action/protocol/context.go

```diff
@@ -64,6 +64,10 @@ type (
 		GasLimit uint64
 		// Producer is the address of whom composes the block containing this action
 		Producer address.Address
+		// AccumTips is the accumulated tips of the block
+		AccumulatedTips big.Int
+		// PrevBaseFee is the base fee of the previous block
+		PrevBaseFee *big.Int
```

Comment:
I think this is base fee of this block, but not last block.
If it is `PrevBaseFee`, we may put it in `BlockchainCtx.Tip`